src/charm.py

```diff
@@ -294,22 +338,25 @@ def _should_run_setup(self, current_plan, s3info: S3Info):
         )
 
     def _are_db_relations_ready(self) -> bool:
-        if not self._stored.db_name:
+        """Check if the needed database relations are established.
+
+        Returns:
+            If the needed relations have been established.
+        """
+        if not self._stored.db_name:  # type: ignore
```

Comment:
Here as well? Usually it is worthwhile to include a comment saying why a issue reported by `mypy` is not correct, same comment applies for the other `type: ignore`

Reply:
Done